### ISSUE TITLE
Fuel not loaded correctly in tanks when using simbrief

### ISSUE THREAD

**Issue by ghifede89** [flybywiresim/a32nx#3102]
**Mod Version**
{
  "built": "2021-01-17T21:53:33+00:00",
  "ref": "",
  "sha": "",
  "actor": "Andreas Guther <andreas@guther.family>",
  "event_name": "manual"
}

**Describe the bug**
When loading fuel from simbrief I noticed that fuel is imported correctly in AOC menu ("PERF/W&B") but once I press the refuel load button it is not transferred correctly in the fuel tanks.
Usually I see approx 70/80 kg less in fuel tanks.

**To Reproduce**
1. configure a flight in simbrief
2. Open AOC menu
3. press OFP request (send)
4. notice fuel value is the same as in simbrief
5. press "refuel" load button
6. notice that FOB is less than what was in simbrief

**Expected behavior**
Fuel in perf/W&B page and FOB should be the same

**Actual behavior**
FOB is 70/80 kgs less than expected

**References**
![Screenshot (201)](https://user-images.githubusercontent.com/76997018/104926854-f2ea1a80-59a0-11eb-888f-32eae767a541.png)
![Screenshot (198)](https://user-images.githubusercontent.com/76997018/104926872-fb425580-59a0-11eb-9dc6-c88299172292.png)
![Screenshot (166)](https://user-images.githubusercontent.com/76997018/104926885-02696380-59a1-11eb-9a0a-4f828b7c0a0b.png)

**Additional context**
I tried searching the issues section for a similar problem but unfortunately did not find anything like this.

Was this working before/when did the issue start occurring?
Noticed this problem a few days ago so the actual mod version is not the incriminated one :-)

**Comment by fradesa84** [comment]
I have the same problem

**Comment by beheh** [comment]
This might have to do with the "unusable" fuel amount that always remains in the tanks. Maybe the upper ECAM subtracts that, and we don't add it on top when setting the refuel amount.

**Comment by ghifede89** [comment]
I thought about this thing but the 70 or 80 kgs (it is always a random value between 70 and 80) missing are more than the reported unusable fuel declared in those files.

**Comment by MisterChocker** [comment]
In real life if you request 5000kg of fuel, you are not really getting those exact 5 tons of fuel.

**Comment by short1st** [comment]
I am still having this issue. The amount of discrepancy is significant and can't be only explained by the unusable fuel. Also, does the ECAM FOB display only usable fuel, while the AOC perf w/b screen AND the MSFS default fuel screen display total? 
I have a hunch that this somehow relates to issue #3528 , which is about the max enterable fuel in the AOC fuel loading field being multiple thousands of kilos higher than the actual max fuel capacity.

**Comment by short1st** [comment]
Update:  The Gross Weight on the ECAM is indeed incremented by the amount of kilograms of fuel added through the AOC menu (or through the MSFS fuel window, for that matter). It seems to only be the FOB display on ECAM which is off.

**Comment by felixharnstrom** [comment]
I don't think this relates to #3528, that's just the MCDU allowing you to load more fuel that should be possible. This was due to a hard-coded value in the MCDU code and should not affect the FOB value displayed on the ECAM.

I think the behavior described in this issue can be explained by a combination of two factors:
1. Usable vs unusable fuel
2. FOB value on lower ECAM always rounding down

Here I added some logging to the ECAM display:
![insim_fuel_10000l_with_fob_log](https://user-images.githubusercontent.com/5500848/107875887-b12a9200-6ec2-11eb-94d6-713ca933a445.PNG)
Note that the ECAM displays 9920 KG FOB, but logging the underlying simvar (FUEL TOTAL QUANTITY) shows the 'true' FOB to be ~9933 KG (you might need to zoom in a a bit to see the log in the upper right corner). This is because the ECAM rounds _down_ to the nearest multiple of 20 (unfortunately I don't have a reference at hand for this behavior). If we then also add the unusable fuel (65.9 KG according to #2286) we get pretty darned close to the 10000 KG displayed in MSFS.

**Comment by sidnov** [comment]
This page has been removed from the MCDU with the last W&B update
